cypress/cypress/support/auth-provider-commands/keycloak.ts

```diff
@@ -40,7 +39,7 @@ const loginToKeycloak = (baseUrl: string, username: string, password: string) =>
         {
             validate: () => {
                 cy.wrap(sessionStorage)
-                    .invoke('getItem', 'oidc.user:https://ec2-52-55-68-28.compute-1.amazonaws.com/realms/mlspace:web-client')
+                    .invoke('getItem', `oidc.user:${keyCloakUrl}realms/mlspace:web-client`)
```

Comment:
Realm and client should probably be configurable similar to URL. Doesn't need to change at the moment though as we're not currently including keycloak with deployments.